changes for fast navigation

Jwsxs opened the pull request.

Jwsxs committed:
Changes to selected music for huge playlists

Pressing Ctrl + down or up will now select the 10th next music, for fast interaction between huge playlists and your time 😅.

Jwsxs committed:
Changes for fast navigation

Intended for large playlists, pressing Ctrl+Up/Down will move 10 lines, and also loops correctly on last and first line

aome510 commented on spotify_player/src/config/keymap.rs:
```diff
@@ -236,6 +236,10 @@ impl Default for KeymapConfig {
                     command: Command::SelectNextOrScrollDown,
                 },
+                Keymap {
+                    key_sequence: "C-down".into(),
+                    command: Command::SelectNextOrScrollDown10,
```
why don't you just bind this key sequence to `PageSelectNextOrScrollDown` will do scroll down of `page_size_in_rows` (default to be `20`) instead of adding a new command

Jwsxs replied:
wasn't sure on how to check the Ctrl command if not that way... also it's 10 for better visibility on doing so, but yeah I'll check that out

Jwsxs committed:
removed unnecessary commands

removed the two commands added, now using `PageSelectPreviousOrScrollUp` and `PageSelectNextOrScrollDown` for fast navigation, looping only on single line movement in case it's used on something else

Jwsxs committed:
Merge branch 'master' into master

Jwsxs committed:
Merge branch 'aome510:master' into master
